pyproject.toml

```diff
@@ -34,7 +35,6 @@ dependencies = [
     "psycopg~=3.1.10",
     "pysftp~=0.2.9",
     "requests>=2.26.0,<3",
```

Comment:
consider placing this in a separate PR, removing this requirement is not linked to the pydantic update